File: idea-plugin/src/main/java/ru/mipt/acsl/decode/parser/psi/DecodeNamedElementImpl.java
package ru.mipt.acsl.decode.parser.psi;

import com.intellij.extapi.psi.ASTWrapperPsiElement;
import com.intellij.lang.ASTNode;
import org.jetbrains.annotations.NotNull;

/**
 * @author Artem Shein
 */
public abstract class DecodeNamedElementImpl extends ASTWrapperPsiElement implements DecodeNamedElement
{
    public DecodeNamedElementImpl(@NotNull ASTNode node)
    {
        super(node);
    }
}

File: parser/src/main/scala/ru/mipt/acsl/decode/parser/psi/DecodeNamedElementImpl.scala
package ru.mipt.acsl.decode.parser.psi

import com.intellij.extapi.psi.ASTWrapperPsiElement
import com.intellij.lang.ASTNode

/**
 * @author Artem Shein
 */
abstract class DecodeNamedElementImpl(node: ASTNode) extends ASTWrapperPsiElement(node) with DecodeNamedElement
